pkg/logs/sender/sender.go

```diff
@@ -84,7 +84,7 @@ func (s *Sender) send(payload *message.Message) {
 			// FIXME: run all `Send` in parallel to avoid the effect on a slow
```

Comment:
You can remove, update the comment I think

Reply:
👍